### ISSUE TITLE
Search bar not clearly visible on App Page

### ISSUE THREAD

**Issue by kanakpandit17** [calcom/cal.com#8693]
The search bar is not clearly visible immediately on user's onboarding on that webpage. This is due to its small size.Also the focus ring towards the right is not visible. 
 
![search_bar](https://user-images.githubusercontent.com/86275069/236435639-6bc87824-8f23-4902-a721-56a843b01cff.jpg)
![search_bar2](https://user-images.githubusercontent.com/86275069/236435659-cf76991a-5c5d-43d2-b61e-787df9140cf5.jpg)

### Steps to Reproduce

1. Go to Apps

**Comment by kanakpandit17** [comment]
I am self assigning this

**Comment by kanakpandit17** [comment]
-

**Comment by Tonie-NG** [comment]
@kanakpandit17  Any reason why this issue was reopened?

**Comment by CarinaWolli** [comment]
Fixed by https://github.com/calcom/cal.com/pull/8741
